WorkerApi/ArmoniK.DevelopmentKit.WorkerApi/Services/ComputerService.cs

```diff
@@ -155,8 +155,13 @@ public override async Task<Output> Process(ITaskHandler taskHandler)
       Logger.LogError(ex,
                       "WorkerAPIException failure while executing task");
 
-      throw new RpcException(new Status(StatusCode.Aborted,
-                                        ex.Message + Environment.NewLine + ex.StackTrace));
+      output = new Output
```

Comment:
I wonder if a return here wouldn't be better ?

Reply:
Done